openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -1,6 +1,7 @@
 """
 Course API Views
 """
+from operator import truediv
```

Comment:
What is this used for?

Reply:
Nothing. It's a typo on my code editor and it will add this. It's now removed.